src/ensembl/src/content/app/custom-download/state/attributes/attributesReducer.ts

```diff
@@ -0,0 +1,105 @@
+import { ActionType, getType } from 'typesafe-actions';
+
+import { RootAction } from 'src/objects';
+import * as attributesActions from './attributeActions';
+import {
+  AttributesAccordionState,
+  defaultAttributesAccordionState,
+  OrthologueState,
+  AttributeContentState
+} from './attributesState';
+
+function attributesAccordion(
+  state: AttributesAccordionState = defaultAttributesAccordionState,
+  action: ActionType<RootAction>
+): AttributesAccordionState {
```

Comment:
`AttributesAccordionState` is the name that remains from the time when these files lived in containers/accordion. Should be renamed to `AttributesAccordionState`.

Reply:
Renamed it to `AttributesState`.